xlr/converters/src/ts-to-xlr.ts

```diff
@@ -469,7 +469,7 @@ export class TsConverter {
       const syntheticType = this.context.typeChecker.typeToTypeNode(
         effectiveType,
         node,
-        undefined
+        1
```

Comment:
Can we use the enum here?

`NodeBuilderFlags.NoTruncation`

Reply:
Ah, good call.